tests/filecheck/dialects/seq/seq_invalid.mlir

```diff
@@ -0,0 +1,8 @@
+// RUN: xdsl-opt --verify-diagnostics --split-input-file %s
+
+builtin.module {
+  %clk = "test.op"() : () -> (!seq.clock)
+
+  // expected-error@+1 {{Operation does not verify: divider value 5 is not a power of 2}}
```

Comment:
I haven't seen this expected-error syntax, where is that from? Is it supported by our filecheck?

```suggestion
  // CHECK: {{Operation does not verify: divider value 5 is not a power of 2}}
```

Reply:
It’s copied from CIRCT, might be wrong. `lit` didn’t complain but can update

Reply:
hmm, I think `filecheck` has the `--check-prefix FOO` but it always appends `CHECK` to whatever you provide it, so I don't think the above prefix triggers anything.